vllm/v1/attention/backends/flex_attention.py

```diff
@@ -774,6 +775,10 @@ def get_kernel_options(query, block_m, block_n,
     kernel_options: dict[str, Union[int, bool]] = {
         "FORCE_USE_FLEX_ATTENTION": True,
     }
+    kernel_options["BLOCK_M"] = 16
+    kernel_options["BLOCK_N"] = 16
+    kernel_options["IS_DIVISIBLE"] = False
```

Comment:
Is there a reason you needed to do this ? whether we are masking loads or not shouldn't affect reduction order.